### PR TITLE
Singular Field Names

### PR DESCRIPTION
**Pull request by dehume**
Keep TF field names singular

### CODE REVIEW

File: integration/connection.tf [hunk 1]
```diff
@@ -34,8 +34,8 @@ resource "materialize_connection_kafka" "kafka_conn_multiple_brokers" {
     database_name = materialize_secret.kafka_password.database_name
     schema_name   = materialize_secret.kafka_password.schema_name
   }
-  sasl_mechanisms = "SCRAM-SHA-256"
-  progress_topic  = "progress_topic"
+  sasl_mechanism = "SCRAM-SHA-256"
+  progress_topic = "progress_topic"
```
Comment (bobbyiliev) [inline integration/connection.tf:38]: I think that this is how we have it in the docs:

https://materialize.com/docs/sql/create-connection/#kafka-auth

It only accepts 1 parameter but is plural

File: pkg/resources/resource_source_kafka.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ var sourceKafkaSchema = map[string]*schema.Schema{
 		Optional:    true,
 		ForceNew:    true,
 	},
-	"include_headers": {
+	"include_header": {
```
Comment (bobbyiliev) [inline pkg/resources/resource_source_kafka.go:48]: Same as the sasl mechanisms, in the docs it is `INCLUDE HEADERS`:

https://materialize.com/docs/sql/create-source/kafka/#headers

**Comment by dehume** [comment]
Reverted the two you mentioned, so now the only change is the nested `column` (which is closer to the original objective of the issue). All the other changes are just terraform formatting.

**Review by bobbyiliev — APPROVED** [review]
LGTM!

**Pull request merged**